test/util/testnode/full_node_test.go

```diff
@@ -132,3 +133,12 @@ func (s *IntegrationTestSuite) Test_FillBlock_InvalidSquareSizeError() {
 		})
 	}
 }
+
+// Test_defaultAppVersion tests that the default app version is set correctly in
+// testnode.
```

Comment:
[Question] Does `testnode` refer to a particular struct in the testnode package? if not, then it is a bit unclear what `the default app version is set correctly in testnode.` means

Reply:
3cf0697b18b10c7f36364767f9170336964c70b0